examples/pip_parse_vendored/requirements.bzl

```diff
@@ -35,6 +35,21 @@ def dist_info_requirement(name):
 def entry_point(pkg, script = None):
     if not script:
         script = pkg
+
+    # buildifier: disable=print
+    print("""
+DEPRECATED: please replace this instance of entry_point with the following:
```

Comment:
I love that you put in a depreciation notice! Can we include a link to the docs?

Reply:
Will do once we have agreed on design.